src/decays/decay_functions.cpp

```diff
@@ -303,4 +303,21 @@ unsigned int number_of_active_flavours(softsusy::QedQcd const& qedqcd, double m)
    return nf;
 }
 
+/*
+ * Computes 1-loop QCD correction to t -> H^+ b + X in the limit mb -> 0
+ * as given in Eq. 2.5 of 2201.08139
+ */
+double Delta_t(double z) noexcept {
+   const double lnz = std::log(z);
+   const double ln1mz = std::log1p(-z);
+   return -2.*Li2(z) + z/(z-1.)*lnz + (1./z - 2.5)*ln1mz - lnz*ln1mz - 2.*zeta2 + 2.25;
```

Comment:
Please catch the divergent cases where `z == 1` etc.

Reply:
Good catch. There is no limit for $z\to1$ but this is ok since $z=1$ is unphysical anyway. It means that the mass of the decaying particle is equal to the mass of the decays products. For $z\to0$ it's slightly more complicated. The limit exists, and equals $\lim_{z\to0} \Delta_t(z) = 5/4 - \pi^2/3$. However, you cannot make a 'real' Taylor series around $z=0$ since it lies outside of the radius of converge of series for $\ln z$. You can write it as partial series though, living logs unexpanded.  I've implemented this in 7d10b85.

Reply:
Great, thanks!